frontend/src/assets/css/typography-variables.css

```diff
@@ -0,0 +1,74 @@
+:root {
```

Comment:
use 16px as the base size. And all other sizes try to extend from it
the same for line-height

Reply:
Done in commit [1453a79](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/1453a7989a7243d01e5231c26d0ce80bef572bfa)